### PR TITLE
Patch for RHEL-07-040520, Patch and Audit for RHEL-07-021120

### PR DESCRIPTION
**Pull request by cgrimm-redhat**

### CODE REVIEW

File: tasks/fix-cat2.yml [hunk 1]
```diff
@@ -649,7 +649,9 @@
       - RHEL-07-021110
 
 - name: "MEDIUM | RHEL-07-021120 | PATCH | If the cron.allow file exists it must be group-owned by root."
-  command: "true"
+  file:
+      path: /etc/cron.allow
+      group: root   
   tags:
       - cat2
       - medium
```
Comment (johannes-cabal) [inline tasks/fix-cat2.yml]: Add cron tag

File: tasks/fix-cat2.yml [hunk 2]
```diff
@@ -649,7 +649,9 @@
       - RHEL-07-021110
 
 - name: "MEDIUM | RHEL-07-021120 | PATCH | If the cron.allow file exists it must be group-owned by root."
-  command: "true"
+  file:
+      path: /etc/cron.allow
+      group: root
```
Comment (johannes-cabal) [inline tasks/fix-cat2.yml:657]: Need to use your audit task from above. You stat for the file, but it fails if the file does not exist.

**Comment by cgrimm-redhat** [comment]
Updated to resolve missing variable and retested.

**Review by johannes-cabal — APPROVED** [review]
Thanks for the fixes.

**Pull request merged**